lib/build/tasks.js

```diff
@@ -38,9 +39,16 @@ const libSassOptions = {
 function generateAssets () {
   clean()
   sassExtensions()
+
+  process.stdout.write('compiling CSS...')
```

Comment:
I know it wasn't introduced here, but shall we use console.log for consistency?

Reply:
We want to write without a newline character, so can't use `console.log`